automation/test.sh

```diff
@@ -90,6 +90,8 @@ elif [[ $TARGET =~ sig-compute-parallel ]]; then
   export KUBEVIRT_PROVIDER=${TARGET/-sig-compute-parallel/}
 elif [[ $TARGET =~ sig-compute ]]; then
   export KUBEVIRT_PROVIDER=${TARGET/-sig-compute/}
+elif [[ $TARGET =~ sig-compute-conformance ]]; then
+   export KUBEVIRT_PROVIDER=${TARGET/-sig-compute-conformance/}
```

Comment:
+1 @fossedihelm  - this will have to be moved above line 91 or it won't get set. 

What is the plan for running these tests in parallel or serial?

Reply:
Ah, I see, for whatever reason I thought is needs to be suffix match. The plan is to run only parallel.